Yoast/wordpress-seo
# Deprecate og image

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* I also deleted `presenter-interface.php` because this interface was never used.

## Test instructions

This PR can be tested by following these steps:

* In the backend, create a post with an image.
* In the frontend, see the meta tag `og:image`.
* Check out this branch, and notice that the meta tag `og:image` has disappeared.
* There should also not be any image-related deprecation notices in the frontend.

## UI changes

## Documentation

## Quality assurance


Fixes #
